neuro_data/pipeline_management/NeuroDataPipelineManagement.py

```diff
@@ -0,0 +1,181 @@
+import numpy as np
```

Comment:
can you choose a better name for this file: `from neuro_data.pipeline_management import NeuroDataPipelineManagement` seems redundant, also module names should be lowercase and usually one word (two are alright too)

Reply:
I am following more of the C++ where the module is an actual class

Reply:
Wait I read that wrong, what name would you suggest then?

Reply:
maybe base. I don't know exactly what you intend for this file to do. Also, maybe this filed should go under neuro_data/static_images as they only apply to static_images (and maybe there it could be called pipeline_management.py)

Reply:
Eventually I will add in the dynamic side too